### PR TITLE
[TTAHUB-1487] Add submitted date for approvers to see

### PR DESCRIPTION
**Pull request by AdamAdHocTeam**
## Description of change

When reviewing a report for approval. We want the approver to see the submitted date if its available.

## How to test

Submit a report for approval. When reviewing the report the they should see the submission date.
If there is no submission date we hide it.

**Mockup:**
https://www.figma.com/file/7lFZynR1T97cfJbCIgBHD4/AR-date-submitted?node-id=1%3A2124&t=W6NMP4c5QWXV4G6S-0

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1487

### Every PR

- [x] Meets issue criteria
- [x] JIRA ticket status updated
- [x] Code is meaningfully tested
- [x] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [ ] API Documentation updated
- [ ] Boundary diagram updated
- [ ] Logical Data Model updated
- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions
- [x] UI review complete

### Production Deploy

- [ ] Staging smoke test completed

### After merge/deploy

- [x] Update JIRA ticket status

### CODE REVIEW

File: frontend/src/pages/ActivityReport/Pages/Review/Approver/Review.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import React, { useContext } from 'react';
+import moment from 'moment-timezone';
```
Comment (thewatermethod) [inline frontend/src/pages/ActivityReport/Pages/Review/Approver/Review.js:2]:
```suggestion
import moment from 'moment';
```

File: frontend/src/pages/ActivityReport/Pages/Review/Approver/Review.js [hunk 2]
```diff
@@ -80,24 +82,35 @@ const Review = ({
...
-            className="margin-bottom-3"
-          >
-            <Dropdown
-              id="status"
+          <>
+            {
+            dateSubmitted
+              ? (
+                <Fieldset className="smart-hub--report-legend margin-top-3" legend="Date Submitted">
+                  <Label className="margin-top-1" htmlFor="date_submitted">{formattedDateSubmitted}</Label>
```
Comment (thewatermethod) [inline frontend/src/pages/ActivityReport/Pages/Review/Approver/Review.js:90]: fieldset/legend are not needed ([they are really only for grouping related controls](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/fieldset), I.E. a group of radio buttons or checkboxes)

Comment (thewatermethod) [reply frontend/src/pages/ActivityReport/Pages/Review/Approver/Review.js:90]: also a label is really just for a form control - I think you just want two paragraph tags here

**Comment by thewatermethod** [comment]
I see you have margin-top-3 on here, which is what the mockup has but the actual spacing doesn't match, since the legend also has margin-top-3. I think whatever markup you eventually land on, the actual distance between the two controls should be margin-top-3/24px

Screenshot 2023-03-06 at 4 38 48 PM

Screenshot 2023-03-06 at 4 40 33 PM

Screenshot 2023-03-06 at 4 40 19 PM

File: frontend/src/pages/ActivityReport/Pages/Review/Approver/Review.js (hunk 1 above)
Comment (AdamAdHocTeam) [reply frontend/src/pages/ActivityReport/Pages/Review/Approver/Review.js]: Updated.

File: frontend/src/pages/ActivityReport/Pages/Review/Approver/Review.js (hunk 2 above)
Comment (AdamAdHocTeam) [reply frontend/src/pages/ActivityReport/Pages/Review/Approver/Review.js:91]: Made both labels.

**Comment by AdamAdHocTeam** [comment]
> I see you have margin-top-3 on here, which is what the mockup has but the actual spacing doesn't match, since the legend also has margin-top-3. I think whatever markup you eventually land on, the actual distance between the two controls should be margin-top-3/24px
> 
> Screenshot 2023-03-06 at 4 38 48 PM
> 
> Screenshot 2023-03-06 at 4 40 33 PM
> 
> Screenshot 2023-03-06 at 4 40 19 PM

I think this is correct now.

image

**Comment by kryswisnaskas** [comment]
Looks good! The only one aspect of this I am curious about is whether Experience/UX and the customer was ok with the submitted date really being the last submitted date. This is likely not an issue if the reports are approved fast, but let's say someone unlocks an approved report couple of months later to fix one typo. Do we care about the original submitted date?

File: frontend/src/pages/ActivityReport/Pages/Review/Approver/Review.js (hunk 2 above)
Comment (thewatermethod) [reply frontend/src/pages/ActivityReport/Pages/Review/Approver/Review.js:91]: a label is really just for a form control - I think you just want two paragraph tags here

**Comment by AdamAdHocTeam** [comment]
> Looks good! The only one aspect of this I am curious about is whether Experience/UX and the customer was ok with the submitted date really being the last submitted date. This is likely not an issue if the reports are approved fast, but let's say someone unlocks an approved report couple of months later to fix one typo. Do we care about the original submitted date?

This logic was ironed out in the JIRA that added the submitted date. I think they want it to be the last time it was submitted.

File: frontend/src/pages/ActivityReport/Pages/Review/Approver/Review.js [hunk 3]
```diff
@@ -80,24 +82,36 @@ const Review = ({
...
-          >
-            <Dropdown
-              id="status"
+          <>
+            {
+            dateSubmitted
+              ? (
+                <>
+                  <p className="source-sans-pro text-bold margin-top-3 margin-bottom-0" htmlFor="date_submitted">Date Submitted</p>
+                  <p className="margin-top-0" htmlFor="date_submitted">{formattedDateSubmitted}</p>
```
Comment (thewatermethod) [inline frontend/src/pages/ActivityReport/Pages/Review/Approver/Review.js:91]: remove those htmlFors and we should be good to go

**Review by kryswisnaskas — APPROVED** [review]
👍

**Review by thewatermethod — APPROVED** [review]

**Pull request merged**